app/views/govuk_publishing_components/components/_machine_readable_metadata.html.erb

```diff
@@ -17,7 +17,7 @@
 <meta property="og:site_name" content="GOV.UK" />
 <meta property="og:type" content="article" />
 <meta property="og:url" content="<%= page.canonical_url %>" />
-<meta property="og:title" content="<%= page.title %>" />
+<meta property="og:title" content="<% if page.content_item['withdrawn_notice'].present? %>[Withdrawn] <% end %><%= page.title %>" />
```

Comment:
I think it's worth adding this logic into [the presenter](https://github.com/alphagov/govuk_publishing_components/blob/master/lib/govuk_publishing_components/presenters/machine_readable/page.rb). Perhaps a method like `page.withdrawn?`?

Reply:
I think it would be good to be able to use the same logic in the [meta tags presenter](https://github.com/alphagov/govuk_publishing_components/blob/master/lib/govuk_publishing_components/presenters/meta_tags.rb#L35) too, but then there are scoping issues — I'm not sure if there's somewhere I could put it where it's in scope for both?